backend/src/tasks/lightning/node-sync.service.ts

```diff
@@ -418,8 +419,18 @@ class NodeSyncService {
     }
   }
 
-  private utcDateToMysql(dateString: string): string {
-    const d = new Date(Date.parse(dateString));
+  /** Decodes a channel id returned by lnd as uint64 to a short channel id */
+  private toShortId(id: string): string {
+    const n = BigInt(id);
+    return [
+      n >> 40n, // nth block
+      (n >> 16n) & 0xffffffn, // nth tx of the block
+      n & 0xffffn // nth output of the tx
+    ].join('x');
+  }
+
+  private utcDateToMysql(date?: number): string {
```

Comment:
I still don't understand what's that function for. `lnd` returns timestamp that we can directly manipulate and insert into the db without ever the need to convert it to string. @softsimon 

We can use `FROM_UNIXTIME(timestamp)` to insert in db.

Reply:
Actually nevermind I'll clean this up when I make the clightning PR